src/strategies/babywealthyclub-multi/index.ts

```diff
@@ -0,0 +1,56 @@
+import { getProvider, getSnapshots } from '../../utils';
+import strategies from '..';
+
+export const author = 'claibornej';
+export const version = '0.1.0';
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const promises: any = [];
+  const blocks = await getSnapshots(
+    network,
+    snapshot,
+    provider,
+    options.strategies.map((s) => s.network || network)
+  );
+
+  for (const strategy of options.strategies) {
+    // If snapshot is taken before a network is activated then ignore its strategies
+    if (
+      options.startBlocks &&
+      blocks[strategy.network] < options.startBlocks[strategy.network]
+    ) {
+      continue;
+    }
+
+    promises.push(
+      strategies[strategy.name].strategy(
+        space,
+        strategy.network,
+        getProvider(strategy.network),
+        addresses,
+        strategy.params,
+        blocks[strategy.network]
+      )
+    );
+  }
+
+  const results = await Promise.all(promises);
+  const resp = results.reduce((finalResults: any, strategyResult: any) => {
+    for (const [address, value] of Object.entries(strategyResult)) {
+      if (!finalResults[address]) {
+        finalResults[address] = 0;
+      }
+      finalResults[address] += value;
+    }
+    return finalResults;
```

Comment:
Hi @claibornej I think you can use multiple strategies in your space than a separate strategy, the result of multiple strategies will be sum of them

Reply:
I need to query the nft quantity of multiple chains

Reply:
@claibornej You can do same from your space settings, you can select different network for different strategy